src/stores/businessStore.ts

```diff
@@ -197,6 +197,26 @@ export const useBusinessStore = defineStore('business', {
       )
     },
 
+    /** Is True if entity is a Continue In Benefit Company. */
+    isCbenCompany (): boolean {
+      return (this.getLegalType === CorpTypeCd.BEN_CONTINUE_IN)
+    },
+
+    /** Is True if entity is a Continue In Ltd Company. */
+    isCCompany (): boolean {
+      return (this.getLegalType === CorpTypeCd.CONTINUE_IN)
+    },
+
+    /** Is True if entity is a Continue In Community Contribution Company. */
+    isCccCompany (): boolean {
+      return (this.getLegalType === CorpTypeCd.CCC_CONTINUE_IN)
+    },
+
+    /** Is True if entity is a Continue In ULC Company. */
+    isCulcCompany (): boolean {
+      return (this.getLegalType === CorpTypeCd.ULC_CONTINUE_IN)
+    },
+
```

Comment:
Added these but only used in unit tests for now. We may need these in future.

Reply:
Let's be more verbose when naming all these new ones.

How about "isContinuationInUlc", etc?

Or "isUlcContinueIn", etc?